ccip/config/evm/Simulated.toml

```diff
@@ -1,5 +1,5 @@
 ChainID = '1337'
-FinalityDepth = 10
+FinalityDepth = 1
```

Comment:
Is this intentional?

Reply:
Yes. Should have been set to 1 from the beginning since the simulated chain actually does have instant finality.